### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by Leon-Li-1** [KaiqiangXiong/CL-MVSNet#12]
> These training logs were not retained. I recall that when it finally converged, the percentage of the pixels with over 2mm depth error was around 0.256-0.260. We will determine whether it has converged based on this metric on the validation set. 

 _Originally posted by @KaiqiangXiong in [#11](https://github.com/KaiqiangXiong/CL-MVSNet/issues/11#issuecomment-2878867624)_

I have trained this model three times. On the test set, the error of 2mm finally converged to around 0.570, not reaching the range of 0.256-0.260.  My training environment consists of a Nvidia 4090 GPU with a batch_size setting of 1. All other parameters are set according to the parameters specified in the source code of GitHub.

**Comment by KaiqiangXiong** [comment]
Our experiments were trained on eight V100 GPUs, with a total batch size of 8. We did not explore other experimental configurations, and different GPUs may affect the results.

**Comment by Leon-Li-1** [comment]
Trank you. I got it.



---- Replied Message ----
| From | Kaiqiang ***@***.***> |
| Date | 10/09/2025 10:30 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [KaiqiangXiong/CL-MVSNet] About training (Issue #12) |
KaiqiangXiong left a comment (KaiqiangXiong/CL-MVSNet#12)

Our experiments were trained on eight V100 GPUs, with a total batch size of 8. We did not explore other experimental configurations, and different GPUs may affect the results.
